src/constants/styleHelpers.ts

```diff
@@ -20,3 +20,17 @@ export const SQUARE_NARROW = {
   width: 20,
   height: 20
 };
+
+export const FIELD = {
+  width: 240,
+  maxWidth: '80%',
```

Comment:
Are these two styles necessary? Use static `width` or `maxWidth`

Reply:
I think just `width: 240` works well. I'm gonna fixed in future PR.